src/metro/devices/display/fast_plot.py

```diff
@@ -15,7 +15,7 @@
 
 try:
     import lttbc
-except ImportError:
+except (ImportError, AttributeError):
```

Comment:
Could you remove catching the `AttributeError`? I'm not a fan of silently suppressing this silently. I will try with a more recent release of `lttbc` that should be built against the proper numpy version.